src/apps/HeyGPT/HeyGPT/App.xaml.cs

```diff
@@ -44,78 +105,45 @@ public static T GetService<T>()
-        {
-            // Default Activation Handler
-            services.AddTransient<ActivationHandler<LaunchActivatedEventArgs>, DefaultActivationHandler>();
-
-            // Other Activation Handlers
-
-            // Services
-            services.AddSingleton<ILocalSettingsService, LocalSettingsService>();
-            services.AddSingleton<IThemeSelectorService, ThemeSelectorService>();
-            services.AddTransient<INavigationViewService, NavigationViewService>();
-
-            services.AddSingleton<IActivationService, ActivationService>();
-            services.AddSingleton<IPageService, PageService>();
-            services.AddSingleton<INavigationService, NavigationService>();
-
-            // Core Services
-            services.AddSingleton<ISampleDataService, SampleDataService>();
-            services.AddSingleton<ILoginService, LoginService>();
-            services.AddInfrastructure();
-            services.AddWindowsInfrastructure();
-            services.AddOpenAI();
-
-            // Application Services
-            services.AddSingleton<ICharacterService, CharacterService>();
-            services.AddSingleton<IClipboardContextPolicy, ClipboardContextPolicy>();
-            services.AddSingleton<IClipboardContextService, ClipboardContextService>();
-
-            // Views and ViewModels
-            services.AddTransient<LoginPageViewModel>();
-            services.AddTransient<SettingsViewModel>();
-            services.AddTransient<SettingsPage>();
-            services.AddTransient<DataGridViewModel>();
-            services.AddTransient<DataGridPage>();
-            services.AddTransient<SampleDataViewModel>();
-            services.AddTransient<SampleDataPage>();
-            services.AddTransient<LetsChatViewModel>();
-            services.AddTransient<LetsChatPage>();
-            services.AddTransient<ShellPage>();
-            services.AddTransient<ShellViewModel>();
-            services.AddTransient<HeaderWorkspaceViewModel>();
-            services.AddTransient<FooterWorkspaceViewModel>();
-
-            // Configuration
-            services.Configure<LocalSettingsOptions>(context.Configuration.GetSection(nameof(LocalSettingsOptions)));
-        }).
-        Build();
-
-        UnhandledException += App_UnhandledException;
-    }
-
     private void App_UnhandledException(object sender, Microsoft.UI.Xaml.UnhandledExceptionEventArgs e)
     {
         // TODO: Log and handle exceptions as appropriate.
         // https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.application.unhandledexception.
     }
 
-    protected async override void OnLaunched(LaunchActivatedEventArgs args)
+    protected async override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
     {
         base.OnLaunched(args);
 
-        await App.GetService<IActivationService>().ActivateAsync(args);
+        // NOTE: When the application is first launched, the bootstrapping process will
```

Comment:
This was a struggle to get right... Seems things have changes from a few .NET versions or WindowsAppSDK versions ago.  There is a ton of details here, Program.cs, and ActivationService.cs to explain how we implemented the activation events.